skia/recorder/include/archive.hpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <fstream>
+#include <zlib.h>
+#include <string>
+#include <zip.h>
+#include <vector>
+
+class Archive
+{
+public:
+	Archive(const std::string& archive_name);
+	~Archive();
+
+	static const std::vector<char> read_file(const std::string& filepath);
+	int add_buffer(const std::string& filename,
```

Comment:
Total nitpick here, but the rest of our C++ method names are camel case, can we name these addBuffer, readFile, etc?